Products/CMFPlone/browser/templates/recyclebin.pt

```diff
@@ -0,0 +1,349 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      metal:use-macro="context/main_template/macros/master"
+      i18n:domain="plone">
+<body>
+
+<metal:override fill-slot="top_slot">
+  <tal:block define="dummy python:request.set('disable_border', 1);
+                     dummy python:request.set('disable_plone.leftcolumn', 1);
+                     dummy python:request.set('disable_plone.rightcolumn', 1)" />
+</metal:override>
+
+<metal:main fill-slot="main">
+    <div class="container-fluid px-0">
+        <!-- Header section with icon -->
+        <div class="d-flex align-items-center mb-4">
+            <div class="flex-shrink-0 me-3">
+                <i class="fas fa-recycle text-success" style="font-size: 2rem;"></i>
+            </div>
+            <div>
+                <h1 class="documentFirstHeading mb-0" i18n:translate="">Recycle bin</h1>
+                <div class="documentDescription text-muted" i18n:translate="">
+                    Items deleted from this site are stored here, and can be restored or permanently deleted.
+                </div>
+            </div>
+        </div>
+        
+        <!-- Search and filter form in a card -->
+        <div class="card mb-4">
+            <div class="card-header bg-light d-flex align-items-center">
+                <i class="fas fa-search me-2"></i>
+                <span i18n:translate="">Search and Filter</span>
+            </div>
+            <div class="card-body">
+                <form method="get" tal:attributes="action string:${context/absolute_url}/@@recyclebin"
```

Comment:
This should be a `z3c.form`

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4168/commits/aa2eddd11ff612031fee466f1bc456b3c7c71421